docs/content/getting-started/configure-the-viewer/save-and-load.md

```diff
@@ -3,22 +3,24 @@ title: Save and load viewer configuration files
 order: 2
 ---
 
-If you have made changes to your blueprint and you would like to save or share these changes,
+If you have made changes to the viewer and you would like to save or share them,
 you can do so by saving your blueprint to an `.rbl` file.
 
 ## Saving a blueprint
 
-To save your blueprint, go to the file-menu and choose "Save blueprint…"
-TODO(#5636): save
+To save the currently active blueprint, go to the file-menu and choose "Save blueprint…"
 
-## Loading a blueprint
-
-Once you have saved a blueprint, you can load it again.
+<picture>
+  <img src="https://static.rerun.io/save_blueprint/85644e086ba9cf7fb81cb7ece55b38bef863c755/full.png" alt="">
+</picture>
 
-When loading a blueprint, it must match the "application_id" used by the recording.
+## Loading a blueprint
 
-To load a blueprint, go to the file-menu and choose "Open…" and select the `.rbl` file you would like to load.
-TODO(#5636): open_1
+Once you have saved a blueprint, you can later load it in the viewer. The
+"Open…" menu item is used to load either recordings (`.rrd`) or blueprints
+(`.rbl`). Alternatively, if your OS supports drag-and-drop, you can drag an
```

Comment:
```suggestion
(`.rbl`). Alternatively you can drag an
```

I feel like this goes without saying, similarly to "if you have a mouse or other pointing device, and hands to use it" goes without saying

Reply:
ha, fair... I'm just in the 1% where this caveat is frequently relevant